src/util/troll.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+#include "gta/enums.hpp"
+#include "services/players/player_service.hpp"
+#include "script_global.hpp"
+#include "pointers.hpp"
+
+namespace big::troll
+{
+	inline void set_bounty_on_player(player_ptr target)
+	{
+		const size_t arg_count = 21;
+		int64_t args[arg_count] =
+		{
+            (int64_t)eRemoteEvent::Bounty,    
+            target->id(),
+            1,
+            10000,
```

Comment:
10k bounty is a modder only bounty value, allow for setting a custom value possibly.

Also weird indents.

Reply:
done the value btws, the indents is the default vs tab indent